src/test/java/uk/gov/justice/digital/delius/integration/secure/UserAPITest.java

```diff
@@ -62,18 +63,19 @@ public void retrieveUserDetailsForMultipleUsers() throws JsonProcessingException
                 .statusCode(200)
                 .extract()
                 .body()
-                .as(JsonNode.class);
+                .as(UserDetails[].class);
 
-        Map<String, UserDetails> userDetailsMap = objectMapper.convertValue(userDetails, new TypeReference<>() {});
-        UserDetails jimSnowUserDetails = userDetailsMap.get("JimSnowLdap");
-        UserDetails sheilaHancockUserDetails = userDetailsMap.get("sheilahancocknps");
+        UserDetails jimSnowUserDetails = Stream.of(userDetails).filter(u -> u.getUsername().equals("JimSnowLdap")).findFirst().orElseThrow();
+        UserDetails sheilaHancockUserDetails = Stream.of(userDetails).filter(u -> u.getUsername().equals("sheilahancocknps")).findFirst().orElseThrow();
 
-        assertThat(userDetails.size()).isEqualTo(2);
+        assertThat(userDetails.length).isEqualTo(2);
 
+        assertThat(jimSnowUserDetails).isNotNull();
```

Comment:
why will this ever be null if you are doing an `orElseThrow` above?

Reply:
agreed - done